docs/Changelog.md

```diff
@@ -10762,7 +10762,7 @@ This version of the operator has been available since version 11 of the default
 <dt><tt>kernel_shape</tt> : list of ints</dt>
 <dd>The shape of the convolution kernel. If not present, should be inferred from input W.</dd>
 <dt><tt>output_padding</tt> : list of ints</dt>
-<dd>The zero-padding added to one side of the output. This is also called adjs/adjustment in some frameworks.</dd>
+<dd>Additinal elements added to the side with higher coordinate indexes in the output. Each element in "output_padding" cannot be larger than the stride along the associated axis. By default, this attribute is a zero vector. Note that this attribute doesn't directly affect the computed output values. It only controls the selection of the computed values. This is also called adjs or adjustment in some frameworks.</dd>
```

Comment:
Additional

Reply:
Thanks!

Reply:
How about:
"Additional elements added to the output tensor shape, extending the side with higher coordinates. Each padding value in `output_padding` must be less than or equal to the corresponding stride value. By default, this attribute is a zero vector of size equal to the output tensor rank. Note that this attribute doesn't directly affect the computed output values. It only controls the shape of the computed values. This is also called `adjs` or `adjustment` in some frameworks."

Reply:
I will merge your ideas into the description.